Makefile

```diff
@@ -55,18 +55,26 @@ helm-test:
 helm-lint:
 	cd config/helm && ./testing/lint.sh
 
+create-kind-cluster:
+	kind create cluster --config ./src/testing/kuttl/oneagent/kind.yaml
+
+delete-kind-cluster:
+	kind delete cluster --name kind
 
 kuttl-install:
 	hack/e2e/install-kuttl.sh
 
-kuttl-oneagent: deploy
+kuttl-oneagent: kuttl-check-mandatory-fields deploy
 	kubectl -n dynatrace wait pod --for=condition=ready -l app.kubernetes.io/component=webhook
```

Comment:
Makefile shouldn't be used to prepare environment for tests if framework is able to do it on it's own.
Things like applying CRD, waiting for webhook etc should be defined in kuttl testsuite.

Reply:
The problem is that the framework is not really able to do what we want.

There are 3 options with downsides that I could think of: 
- Run `make deploy` in the testsuit command section. => No option to delete the operator 
- Add manifests to `manifestsDirs` => No way to wait until our operator is running, => first tests will fail due to operator not yet ready
- Add command to every test directory => same code in every test folder. + the operator should be running during the whole test suite

Therefore I would stick to the makefile.

Reply:
@gkrenn 
> * Run `make deploy` in the testsuit command section. => No option to delete the operator

Why is cleanup needed? Kind cluster is deleted after tests is done anyway

> * Add manifests to `manifestsDirs` => No way to wait until our operator is running, => first tests will fail due to operator not yet ready

AG tests contain assert checking if operator is fully operational, see: https://github.com/Dynatrace/dynatrace-operator/blob/master/src/testing/kuttl/activegate/tests/capabilities-walkthrough/10-setup.yaml#L4-L7

> Add command to every test directory => same code in every test folder. + the operator should be running during the whole test suite

Operator (running for whole test suite) can be started using `manifestsDir` from https://kuttl.dev/docs/testing/reference.html#testsuite